### ISSUE TITLE
Calva Format is not working after upgrade to v2.0.97

### ISSUE THREAD

**Issue by andon** [BetterThanTomorrow/calva#640]
Both of the **Calva Format: Format** commands stopped working for me after calva was (auto-upgraded) to v.2.0.97

![calva-format](https://user-images.githubusercontent.com/1130655/81380910-e638d580-9135-11ea-98f2-c1ea761ccf43.png)

When I call them on any top-level form, the show me the following errors:
* Command 'Calva Format: Format and Align Current Form (recursively, experimental)' resulted in an error (Running the contributed command: 'calva-fmt.alignCurrentForm' failed.)
* Command 'Calva Format: Format Current Form' resulted in an error (Running the contributed command: 'calva-fmt.formatCurrentForm' failed.)

As the error happens to me all the time, I don't think there is point to provide example clj project.

The following place might be a potential place where the error is happening:
https://github.com/BetterThanTomorrow/calva/compare/v2.0.96...v2.0.97#diff-5b2054c20efcd52a24589de0610093d6L38

If necessary I can provide my vs-code settings.

```
Version: 1.45.0 (user setup)
Commit: d69a79b73808559a91206d73d7717ff5f798f23c
Date: 2020-05-07T16:18:48.860Z
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
```

**Comment by PEZ** [comment]
Thanks. I can reproduce it. The key here was:

> When I call them on any _top-level_ form

Until fixed, move inside the form before hitting `tab`.

**Comment by PEZ** [comment]
@andon I think I have fixed this in this VSIX: https://6040-125431277-gh.circle-artifacts.com/0/tmp/artifacts/calva-2.0.99-fix-format-top-level-form-640-73fb7955.vsix

Can you give it a spin?

**Comment by andon** [comment]
@PEZ It works. No errors, just formatting :)

Thank you 🙇
